test/e2e/framework/pv_util.go

```diff
@@ -92,124 +91,166 @@ type PersistentVolumeClaimConfig struct {
 }
 
 // Clean up a pv and pvc in a single pv/pvc test case.
-func PVPVCCleanup(c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) {
-	DeletePersistentVolumeClaim(c, pvc.Name, ns)
-	DeletePersistentVolume(c, pv.Name)
+func PVPVCCleanup(c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) error {
+
+	var err error
+	if pvc != nil {
+		if err = DeletePersistentVolumeClaim(c, pvc.Name, ns); err != nil {
+			return err
+		}
+	}
+	if pv != nil {
+		if err = DeletePersistentVolume(c, pv.Name); err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
 // Clean up pvs and pvcs in multi-pv-pvc test cases. All entries found in the pv and
 // claims maps are deleted.
-func PVPVCMapCleanup(c clientset.Interface, ns string, pvols PVMap, claims PVCMap) {
+func PVPVCMapCleanup(c clientset.Interface, ns string, pvols PVMap, claims PVCMap) error {
+
+	var err error
+
 	for pvcKey := range claims {
-		DeletePersistentVolumeClaim(c, pvcKey.Name, ns)
+		if err = DeletePersistentVolumeClaim(c, pvcKey.Name, ns); err != nil {
+			return fmt.Errorf("Failed to delete PVC %q: %v", pvcKey.Name, err)
```

Comment:
Can we print `error:` in the message.
return fmt.Errorf("Failed to delete PVC %q, error: %v", pvcKey.Name, err)